sprocs/instance_questions_next_allowed_grade.sql

```diff
@@ -0,0 +1,63 @@
+DROP FUNCTION IF EXISTS instance_questions_next_allowed_grade(bigint);
+CREATE OR REPLACE FUNCTION
+    instance_questions_next_allowed_grade (
+        IN iq_id BIGINT,
+        OUT allow_grade_date TIMESTAMPTZ,
+        OUT allow_grade_formatted_date TEXT,
```

Comment:
This will be removed, as it's only used in one place and requires additional queries to get the timezone. The computation of this date may be replaced by the interval as a text (e.g., 5 minutes, or 30 seconds).